src/modules/visitor-space/components/MaterialRequestBlade/MaterialRequestBlade.tsx

```diff
@@ -63,6 +67,15 @@ const MaterialRequestBlade: FC<MaterialRequestBladeProps> = ({
 		type || MaterialRequestType.VIEW
 	);
 	const [reasonInputValue, setReasonInputValue] = useState(reason || '');
+	const user = useSelector(selectUser);
+
+	const { refetch: refetchMaterialRequests } = useGetPendingMaterialRequests(
+		{
+			orderProp: MaterialRequestKeys.maintainer,
+			orderDirection: OrderDirection.asc,
+		},
+		{ enabled: !!user && user.groupName !== GroupName.KIOSK_VISITOR }
+	);
```

Comment:
do we still need this, since the error is thrown before we even open this blade?